File: Chat.Identity.Application/Commands/RegisterCommand.cs
using Chat.Framework.CQRS;

namespace Chat.Identity.Application.Commands;

public class RegisterCommand : ICommand
{
    public string FirstName { get; set; }

    public string LastName { get; set; }

    public DateTime BirthDay { get; set; }

    public string Email { get; set; }

    public string Password { get; set; }
}

File: chat-app/src/app/identity/commands/register-command.ts
import { CommandBase } from "src/app/core/models/command-base";
import { Configuration } from "src/app/core/services/configuration";

export class RegisterCommand extends CommandBase {
    public firstName : any;
    public lastName : any;
    public birthDay : any;
    public email : any;
    public password : string | undefined;

    constructor() {
        super();
        this.apiUrl = Configuration.identityApi.concat("/User/register");
    }
}
